web/assets/scss/bundle.scss

```
@@ -1,41 +1,37 @@
 .bundle-card-line {
     width: 100%;
     height: 225px;
-    border-left: 15px solid #2a9dc7;
     background: #fff;
-    box-shadow: 0 7px 15px #ddd;
-    overflow: hidden;
     margin: 0 0 40px 0;
+    border: solid 1px #ccc;
+    border-radius: 5px;
+    -moz-border-radius: 5px;
+    -webkit-border-radius: 5px;
+    padding: 25px 20px;
+    overflow: hidden;
+
+    &:hover {
+        box-shadow: 0 7px 15px #ddd;
+    }
 }
 
 .bundle-card-line-data {
-    width: calc(100% - 70px);
+    height: 185px;
     float: left;
     overflow: hidden;
 }
 
 .bundle-card-line-information {
-    width: 75%;
     float: left;
-    min-height: 180px;
     overflow: hidden;
-    border-right: 1px solid #d3d3d3;
-    padding: 0 45px 0 20px;
 }
 
 .bundle-card-line-information h2 {
-    font-size: 26px;
-    color: #63605d;
-    font-weight: 400;
+    font-size: 19px;
+    color: #333;
+    font-weight: bold;
```

I think it's better to be consistent with the rest of the css and not use `bold`.
```suggestion
    font-weight: 600;
```